README.md

```diff
@@ -81,14 +81,16 @@ v up
 
 ### C compiler
 
-You'll need Clang or GCC or Visual Studio. If you are doing development, you most likely already have one of those installed.
+It's recommended to use Clang or GCC or Visual Studio. If you are doing development, you most likely already have one of those installed.
 
 Otherwise, follow these instructions:
 
 - [Installing a C compiler on Linux and macOS](https://github.com/vlang/v/wiki/Installing-a-C-compiler-on-Linux-and-macOS)
 
 - [Installing a C compiler on Windows](https://github.com/vlang/v/wiki/Installing-a-C-compiler-on-Windows)
 
+However, if none is found when running `make`, TCC would be downloaded and set as an alternative C backend. It's very lightweight (several MB) so this shouldn't take too long.
```

Comment:
That works currently on linux and windows, but not yet on macos I think.

Reply:
Thanks. Changed.

Reply:
@spytheman tcc now compiles macos binaries, but v.c doesn't compile yet unfortunately (getting a strange #include stdio.h error)